src/icons/states/UsersEmptyState.js

```diff
@@ -0,0 +1,125 @@
+// @flow
+import * as React from 'react';
+
+import classNames from 'classnames';
+import AccessibleSVG from '../accessible-svg';
+import { bdlBoxBlue, white } from '../../styles/variables';
+
+type Props = {
+    className?: string,
+    color?: string,
+    height?: number,
+    opacity?: number,
+    /** A text-only string describing the icon if it's not purely decorative for accessibility */
+    title?: string,
+    width?: number,
+};
+
+const UsersEmptyState = ({ className, color = bdlBoxBlue, height = 135, opacity, title, width = 130 }: Props) => (
+    <AccessibleSVG
+        className={classNames('users-empty-state', className)}
+        height={height}
+        title={title}
+        viewBox="0 0 130 135"
+        width={width}
+    >
+        <g fill="none" fillRule="evenodd">
```

Comment:
This `<g>` wrapper is likely redundant.

Reply:
removing <g>  adds a weird shade to the icon
<img width="288" alt="Screen Shot 2019-09-24 at 3 12 06 PM" src="https://user-images.githubusercontent.com/10932475/65554203-b6f29280-dedd-11e9-831a-3a532f0c3eb7.png">